util/chunk/chunk.go

```diff
@@ -146,6 +149,51 @@ func (c *Chunk) AppendRow(colIdx int, row Row) {
 	}
 }
 
+// Append appends rows in [begin, end) in another Chunk to a Chunk.
+func (c *Chunk) Append(other *Chunk, begin, end int) {
```

Comment:
should we check the validation of begin and end first.

Reply:
no need to, all Chunk's API should not do the validation work.